packages/runtime/datastore-definitions/src/dataStoreRuntime.ts

```diff
@@ -41,6 +41,10 @@ export interface IFluidDataStoreRuntime extends
 
     readonly IFluidHandleContext: IFluidHandleContext;
 
+    readonly rootRoutingContext: IFluidHandleContext;
+    readonly ddsRoutingContext: IFluidHandleContext;
+    readonly objectsRoutingContext: IFluidHandleContext;
```

Comment:
nit: root/dds are singular; objects is plural.  for consistency they should match, so probably changing to `objectRoutingContext` in this case

Reply:
Good point, switched to singleton